### PR TITLE
Firebase Setup values

### PR DESCRIPTION
**Pull request by jongbonga**
Retrieve and use values from the setup page

### CODE REVIEW

File: web-client/src/app/state/setup/setup.store.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { Store, StoreConfig } from '@datorama/akita';
+
+export interface SetupState {
+  code: string;
```
Comment (billguo99) [inline web-client/src/app/state/setup/setup.store.ts:5]: need to change this to `tokenSymbol`

File: web-client/src/app/state/setup/setup.store.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  tokenIssuer: string;
+  xrpIssuer: string;
+  ledger: string;
+  name: string;
+}
+
+export const createInitialState = (): SetupState => ({
+  code: '',
```
Comment (billguo99) [inline web-client/src/app/state/setup/setup.store.ts:13]: need to change this to `tokenSymbol`

File: web-client/src/app/state/setup/setup.query.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { SetupState, SetupStore } from './setup.store';
+
+@Injectable({ providedIn: 'root' })
+export class SetupQuery extends Query<SetupState> {
+  tokenIssuer = this.getValue().tokenIssuer.trim() || environment.tokenIssuer;
+  xrpIssuer = this.getValue().xrpIssuer.trim() || environment.xrpIssuer;
+  ledger = this.getValue().ledger.trim()
+    ? { ...environment.xrplClient, server: this.getValue().ledger }
+    : environment.xrplClient;
+
```
Comment (billguo99) [inline web-client/src/app/state/setup/setup.query.ts:13]: Will we add other fields as needed like `name`, `tokenSymbol`? Right now, these 3 values are the only ones that needs to be "configurable"?

**Review by billguo99 — CHANGES_REQUESTED** [review]
Nice!

File: web-client/src/app/state/setup/setup.query.ts (hunk 3 above)
Comment (jongbonga) [reply web-client/src/app/state/setup/setup.query.ts:13]: Correct.
The name is already in the environment so shouldn't be configurable.
The tokenSymbol will probably be used yes

**Review by billguo99 — APPROVED** [review]

**Comment by billguo99** [comment]
LGTM

**Pull request merged**